rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -2145,6 +2146,55 @@ void rskTxWaitingForSignature_uses_pegoutCreation_rskTxHash_after_rskip_375_acti
         assertEquals(0, provider.getReleaseTransactionSet().getEntries().size());
     }
 
+    @Test()
+    void rskTxWaitingForSignature_override_entry_attempt_after_rskip_375_activation() throws IOException {
```

Comment:
We should also have a test pre RSKIP 375 that shows it was possible to override an entry